Ridhwan Ramadhany

bobkombat opened the pull request:
FIREBASE = https://e-commerce-cms-66bdc.web.app/
HEROKU = https://e-commerce-cms-gacor.herokuapp.com/

JWT_SECRET=secret

email = admin@email.com
password = 1234

- search bar
- filter by tag
- filter by stock
- delete confirmation
- route if product or client routing not found

(-) kalo image_url kepanjangan fail buat add new product

bobkombat committed:
- done jest, supertest

Sursev07 commented on client/src/components/HelloWorld.vue:
```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="hello">
```
nanti ini dihapus ya kalau ga dipakai

bobkombat committed:
- add home page '/'
- add login form '/login'
- add client router '/products'

bobkombat committed:
- add search bar
- add filter by tag
- add filter by stock
- add show details
- add more styling
- add product adding form

bobkombat committed:
- add 404 router client for product not found
- add 404 for not any matching router
- add edit option
- add delete option

bobkombat committed:
Update README.md

bobkombat committed:
Update README.md
